stdlib/3/os/__init__.pyi

```diff
@@ -322,7 +322,10 @@ def close(fd: int) -> None: ...
 def closerange(fd_low: int, fd_high: int) -> None: ...
 def device_encoding(fd: int) -> Optional[str]: ...
 def dup(fd: int) -> int: ...
-def dup2(fd: int, fd2: int) -> None: ...
+if sys.version_info >= (3, 7):
+    def dup2(fd: int, fd2: int) -> int: ...
+else:
+    def dup2(fd: int, fd2: int) -> None: ...
```

Comment:
Could you also add the `inheritable` flag while you are here?

Reply:
Oh, good call.